apps/ttd-app/src/components/ProvenanceDrawer.css

```
@@ -0,0 +1,195 @@
+}
+
+.diff-marker {
+  font-weight: bold;
+  width: 12px;
+}
+
+.diff-old pre,
+.diff-new pre {
+  margin: 0;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.2.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)